components/01-globals/icon/_icon.scss

```diff
@@ -0,0 +1,18 @@
+$icon-width       : 48px !default;
+$icon-padding     : 12px !default;
+$icon__svg-width  : 24px !default;
+$icon__svg-height : 24px !default;
+$icon__svg-padding: 2px !default;
+
+.icon {
+    padding: $icon-padding;
+    display: flex;
+    flex-flow: row nowrap;
```

Comment:
why we use here flex?

Reply:
to inline the `svg` insine `i` vertically. I can also use `display: inline-block` and `vertical-align: middle`. But @Igloczek said that we shouldn't use it. I'm not sure which one is better...

Reply:
after our conversation I see that `display: block` is enough. Thanks, corrected :)